app/models/map.rb

```diff
@@ -114,57 +114,57 @@ def nodes
 
   # find all other maps within <dist> degrees lat or lon
   def nearby_maps(dist)
-    if self.lat.to_f == 0.0 || self.lon.to_f == 0.0
-      return []
+    if lat.to_f == 0.0 || lon.to_f == 0.0
+      []
     else
-      return Map.find(:all,:conditions => ['id != ? AND lat > ? AND lat < ? AND lon > ? AND lon < ?',self.id,self.lat-dist,self.lat+dist,self.lon-dist,self.lon+dist], :limit => 10)
+      Map.find(:all, conditions: ['id != ? AND lat > ? AND lat < ? AND lon > ? AND lon < ?', id, lat - dist, lat + dist, lon - dist, lon + dist], limit: 10)
     end
   end
 
   def average_scale
     # determine optimal zoom level
     puts '> calculating scale'
     pxperms = []
-    self.placed_warpables.each do |warpable|
-      pxperms << 100.00/warpable.cm_per_pixel if warpable.placed?
+    placed_warpables.each do |warpable|
+      pxperms << 100.00 / warpable.cm_per_pixel if warpable.placed?
     end
-    average = (pxperms.inject {|sum, n| sum + n })/pxperms.length
+    average = (pxperms.inject { |sum, n| sum + n }) / pxperms.length
     average
   end
 
   def best_cm_per_pixel
-    hist = self.images_histogram
+    hist = images_histogram
     scores = []
-    (0..(hist.length-1)).each do |i|
+    (0..(hist.length - 1)).each do |i|
       scores[i] = 0
-      scores[i] += hist[i-3] if i > 3
-      scores[i] += hist[i-2] if i > 2
-      scores[i] += hist[i-1] if i > 1
+      scores[i] += hist[i - 3] if i > 3
+      scores[i] += hist[i - 2] if i > 2
+      scores[i] += hist[i - 1] if i > 1
       scores[i] += hist[i]
-      scores[i] += hist[i+1] if i < hist.length - 2
-      scores[i] += hist[i+2] if i < hist.length - 3
-      scores[i] += hist[i+3] if i < hist.length - 4
+      scores[i] += hist[i + 1] if i < hist.length - 2
+      scores[i] += hist[i + 2] if i < hist.length - 3
+      scores[i] += hist[i + 3] if i < hist.length - 4
     end
     highest = 0
-    scores.each_with_index do |s,i|
+    scores.each_with_index do |s, i|
       highest = i if s > scores[highest]
     end
     highest
   end
 
   def average_cm_per_pixel
-    if self.warpables.length > 0
+    if !warpables.empty?
       scales = []
       count = 0
       average = 0
-      self.placed_warpables.each do |warpable|
+      placed_warpables.each do |warpable|
         count += 1
         res = warpable.cm_per_pixel
-        res = 1 if res == 0 # let's not ever try to go for infinite resolution
-        scales << res unless res == nil
+        res = 1 if res.zero? # let's not ever try to go for infinite resolution
+        scales << res unless res.nil?
       end
-      total_sum = (scales.inject {|sum, n| sum + n }) if scales
-      average = total_sum/count if total_sum
+      total_sum = (scales.inject { |sum, n| sum + n }) if scales
+      average = total_sum / count if total_sum
```

Comment:
I don't know if it is the case here, but I guess it is worth mentioning. I was taking much longer reviewing PRs at work because of different settings for formatting on save in the code editors, so people kept doing and undoing space changes. If that is the case, it would be great to define a standard, or format code with some other tool :)

Reply:
Rules such as `Layout/SpaceAroundBlockParameters`, `Layout/SpaceAroundEqualsInParameterDefault`, `Layout/SpaceAroundOperators` and more try to define such spacing rules. The thing is that after we merge this we should quickly integrate this with our pipeline to encourage contributors to use our code pattern. Otherwise problems such as you're described will keep happening.